showcase/src/app/demo/table/demo-table.component.scss

```diff
@@ -18,3 +18,39 @@
     margin-bottom: 0;
   }
 }
+
+.even td {
+  background-color: #d1e1ff;
```

Comment:
Regarding all the styles in this SCSS file, I suggest we remove the element selectors like `td` and `th`... otherwise it feels like we are cheating setting directly the styles for specific HTML tags 😆 which might not always be the case if we change our Table component at some point